libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/ComponentInstaller.kt

```diff
@@ -156,6 +155,30 @@ fun ComponentInstaller.recenterButton(
     )
 }
 
+/**
+ * Install component that toggles [NavigationCamera] between [NavigationCameraState.OVERVIEW] and
+ * [NavigationCameraState.FOLLOWING] on [button] click.
+ *
+ * The installed component will first use the [NavigationCamera] instance from [CameraModeConfig].
+ * If not available, it will attempt to use the [NavigationCamera] instance managed by the NavigationCamera component.
+ * (see [ComponentInstaller.navigationCamera])
+ */
+@ExperimentalPreviewMapboxNavigationAPI
```

Comment:
Do you need experimental here and everywhere else in this class?

Reply:
Yes. Refers
> not removing @ExperimentalPreviewMapboxNavigationAPI from any files related to ComponentInstaller and it's usage. ComponentInstaller could be subject to modifications and we need to play more with them before declaring them stable.